<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>940</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>数据对比工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="testButton1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>300</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>测试连接1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testButton2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>300</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>测试连接2</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>360</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>300</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>条件查询</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>21</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>*时间：</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>21</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2016</year>
       <month>10</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>75</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>开始查询</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>195</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>地点：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="addrEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>380</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>全表查询</string>
    </property>
    <widget class="QPushButton" name="searchButton2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>开始查询</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>360</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>200</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
